### ISSUE TITLE
New header in responsive mobile portrait/landscape

### ISSUE THREAD

**Issue by mjancik** [kiwicom/smart-faq#416]
New headers should be implemented in mobile versions to better the usability for our users.
Landscape related issues: #401 #405 
![smallheaderlogin copy 3](https://user-images.githubusercontent.com/37365838/41297024-7f59a638-6e5e-11e8-8cd0-580122984804.png)

The header should disappear when scrolling down: https://zpl.io/VOkXen1

Designs (portrait): https://app.zeplin.io/project/5a8e8992554d385c76ad5aeb/dashboard?seid=5b1fd0955639ebad77152b5f

Designs (lanscape): https://app.zeplin.io/project/5a8e8992554d385c76ad5aeb/dashboard?seid=5b1fd052e02db4652256c1da

**Comment by kantord** [comment]
@edgarordonez this will also close https://github.com/kiwicom/smart-faq/issues/338, correct?

**Comment by edgarordonez** [comment]
Probably yes @kantord, in fact with the new header should load the FAQ before, and the user can press the options to see the information that is showing the Header currently. So, if I understood correctly, I guess that it will close #338.

**Comment by kantord** [comment]
@edgarordonez Will you work on this in the near future? If no, can you unassign?

**Comment by mjancik** [comment]
## Design review:

The header should disappear or be just static when you scroll through the FAQ. https://zpl.io/VOkXen1

After clicking trip it should expand everything, not only the trip heading.
![smart faq demo 2018-07-19 11-08-52](https://user-images.githubusercontent.com/37365838/42933120-45f96d44-8b44-11e8-9b92-27429d7c10d3.png)

Arrow to go back is in the header only when you are in an article, not when you are in a category.
![smart faq demo 2018-07-19 11-14-10](https://user-images.githubusercontent.com/37365838/42937992-1cb6a61a-8b51-11e8-9547-799b7d9841b0.png)

**Comment by kantord** [comment]
:heavy_check_mark: Back button fixed

**Comment by kantord** [comment]
:heavy_check_mark: booking detail expansion fixed

**Comment by kantord** [comment]
:heavy_check_mark: scroll-hide issue fixed

**Comment by MagdalenaAlmanova** [comment]
I found a few things: 
The first one is about scrolling, when I am on an upcoming trip I cannot scroll down. For better understanding, I will send you @kantord video in DM.  

The second thing is about correct centralization of "manage my booking" and "select other.." on the landscape version. 
Demo v. 
![image](https://user-images.githubusercontent.com/37298895/43023643-e50eedd2-8c6b-11e8-987d-ef220c4e51ca.png)
Prototype v. 
![screen shot 2018-07-20 at 22 26 25](https://user-images.githubusercontent.com/37298895/43023678-00022e10-8c6c-11e8-96f0-64134cfd9177.png)

Another thing is about the magnifier: When I click on the magnifier and go on an upcoming trip. The magnifier is still present. Is IT ok?
![image](https://user-images.githubusercontent.com/37298895/43024030-2cd97f82-8c6d-11e8-9ffb-3e6305621e48.png)

The last issue is about Opera mini. There is a problem with login. I will send you a video. :) 

Thanks

**Comment by kantord** [comment]
> Another thing is about the magnifier: When I click on the magnifier and go on an upcoming trip. The magnifier is still present. Is IT ok?

I cannot reproduce this.

**Comment by jalbertsr** [comment]
@kantord This last thing was fixed in https://github.com/kiwicom/smart-faq/pull/575

**Comment by kantord** [comment]
Cool, that's why I cannot reproduce it then :-)

**Comment by jalbertsr** [comment]
after https://github.com/kiwicom/smart-faq/pull/600, landscape design should be ok, check it again pls

**Comment by kantord** [comment]
I think we cannot close this yet because of the scroll issue :(

The opera mini issue is probably not related to this issue however.

**Comment by MagdalenaAlmanova** [comment]
The scroll issue is relatively still persistent. When I go to "select another trip" I cannot scroll down. I will send you video :) 

I go to "select another trip" and how can I go back from this page? When I clicked on magnifier, it nothing happened.

The last thing, when I typed into the search box and scroll down, It looks like this: 
![img_0053](https://user-images.githubusercontent.com/37298895/43861419-0b3bb054-9b57-11e8-9d1c-eaeab06424d9.jpg)

Thank you @jalbertsr @kantord
